README.rst

```diff
@@ -210,6 +210,54 @@ It is not triggered when the device is created for the first time.
 Settings
 --------
 
+``OPENWISP_SSH_AUTH_TIMEOUT``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-----------+
+| **type**:    | ``float`` |
++--------------+-----------+
+| **default**: | `2`       |
```

Comment:
ReStructuredText needs two slant quotes, it should be:

```
``2``
```

Applies to the other settings as well.